cypress.config.js

```diff
@@ -0,0 +1,13 @@
+const { defineConfig } = require('cypress');
```

Comment:
New root file, need to report to Blizzz and Skjnldsv to make sure it's excluded on packaging